go/client/cmd_signup.go

```diff
@@ -122,6 +122,7 @@ func (s *CmdSignup) ParseArgv(ctx *cli.Context) error {
 		s.genPGP = ctx.Bool("pgp")
 		s.genPaper = true
 		s.doPrompt = false
+		s.storeSecret = true
```

Comment:
In standalone mode, you can't stay "logged in" unless secret is stored.

With service, you can until the service restarts...

But secret should always be stored.  This is a bug in signup in general.